rocAL/rocAL/include/decoders/image/fused_crop_decoder.h

```diff
@@ -56,9 +57,10 @@ class FusedCropTJDecoder : public Decoder {
 
     ~FusedCropTJDecoder() override;
     void initialize(int device_id) override {};
-    bool is_partial_decoder() override { return _is_partial_decoder; };
-    void set_bbox_coords(std::vector <float> bbox_coord) override { _bbox_coord = bbox_coord;};
+    bool is_partial_decoder() override { return _is_partial_decoder; }
+    void set_bbox_coords(std::vector <float> bbox_coord) override { _bbox_coord = bbox_coord;}
     std::vector <float> get_bbox_coords() override { return _bbox_coord;}
+    void set_crop_window(CropWindow &crop_wind) override { _crop_window = crop_wind;}
```

Comment:
Please add space after semicolon in lines 60 and 63

Reply:
changed spacing